src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -1,8 +1,47 @@
 /**
  * A sample SIMID ad that shows a map of nearby locations
  */
+const AdParamKeys = {
+  BUTTON_LABEL: 'buttonLabel',
+  SEARCH_QUERY: 'searchQuery',
+  MARKER: 'marker',
+};
+
+const FIND_NEAREST = "Find Nearest ";
+const DEFAULT_BUTTON_LABEL = "Location";
+
 class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    const adParams = JSON.parse(this.creativeData.adParameters);
+    let buttonLabel = adParams[AdParamKeys.BUTTON_LABEL]; 
+    //ToDo(juliareichel@): handle invalid JSON and param errors
+    const searchQuery = adParams[AdParamKeys.SEARCH_QUERY];
+    //ToDo(juliareichel@): handle case where searchQuery is undefined
+    const marker = adParams[AdParamKeys.MARKER];
+    this.specifyButtonFeatures_(buttonLabel);
+  }
+ 
+  /** @private 
```

Comment:
@private should be the last line of the jsdoc
```
/**
 * Function description ending in a period.
 * @param {type} paramName Descirption of the parameter also ending in period.
 * @private
 */
```

Reply:
Good to know